README.md

```diff
@@ -152,6 +158,13 @@ the simplified yang file)
 \<path_to_xml_files> -p \<path_to_yang_modules> -f yang
 ```
 
+* **Analyze module**: To analyze all files in given directory:
+
+```
+./lyv -r -p \<path_to_module_dependencies>
+\<path_to_the_yang_module>
```

Comment:
Does this work for you? You haven't specified the "format" in this command.

Reply:
fixed